specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-04-01/examples/Endpoint-DELETE-External.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "subscriptionId": "e68d4145-c9ae-4667-925d-c51c8d88ad73",
+    "resourceGroupName": "azuresdkfornetautoresttrafficmanager1421",
+    "profileName": "azsmnet6386",
+    "endpointType": "ExternalEndpoints",
+    "endpointName": "azsmnet7187",
+    "api-version": "2017-05-01"
```

Comment:
I think we should verify whether the versions mentioned in these examples match with the version number in the directory path here.